### PR TITLE
Joss-Submission

### PR DESCRIPTION
**Pull request by rlizzo**
@lantiga please find attached my draft paper for submission of VMTK 1.4 to the [Journal of Open Source Software](https://joss.theoj.org/). The current status satisfies the author guidelines: https://joss.theoj.org/about#author_guidelines, and sits at about 750 words. This shouldn't take long, a quick review would be appreciated!

What I need from you:
1) General review of `paper.md` for clarity and wording. 
2) Your ORCID ID from https://orcid.org/ (sorry, I wasn't able to find you there)
3) If you would like anyone else listed as an author on the paper or in the acknowledgments section.

Once you give me your ORCID ID and I get the go ahead to merge, I'll handle submission and manage the review process. Thanks

### CODE REVIEW

File: doc/joss-paper/paper.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+seeds. Surface editing, mesh generation, and geometric characterization is
+largely automated, relying heavily on the centerline definitions calculated
+from segmented structures.  User interaction is facilitated by a unique system
+of unix-inspired ``Pypes``.  This interface allows for the composable creation
+and execution of entire analyses from simple terminal commands, providing a
+flexible framework for high-level coding, both from the user’s and from the
+developer’s point of view.
+
+``VMTK`` is a mature package with an active development team and user
+community.  It can be used via it's standalone interface, included as a Python
```
Comment (lantiga) [inline doc/joss-paper/paper.md]: its

File: doc/joss-paper/paper.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+or C++ library, or as an extension to the medical image processing platform 3D
+Slicer.  It has received 41 citations in scientific publications since its
```
Comment (lantiga) [inline doc/joss-paper/paper.md]: 3D Slicer's ref?

Comment (lantiga) [inline doc/joss-paper/paper.md]: My MBEC 2008 paper (see link) describes vmtk and it has received 396 citations so far. Many papers refer to vmtk through this citation https://scholar.google.it/citations?hl=it&user=cA3i9uEAAAAJ#d=gs_md_cita-d&p=&u=%2Fcitations%3Fview_op%3Dview_citation%26hl%3Dit%26user%3DcA3i9uEAAAAJ%26citation_for_view%3DcA3i9uEAAAAJ%3Au-x6o8ySG0sC%26tzom%3D-120

File: doc/joss-paper/paper.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+path, which is equivalent to finding the shortest paths in the radius metric.
+
+``vmtkcenterlines -ifile foo.vtp -ofile foo_centerlines.vtp``
+
+![centerline generation](centerlines.png)
+
+The input surface representation (left), a visualization of the internal subset
+of the voronoi diagram where each sheet represents a maximum inscribed sphere
+radius centered at some point in the surface (center), the centerline extracted
+from the voronoi diagram rendered as in it's position within the input surface
```
Comment (lantiga) [inline doc/joss-paper/paper.md]: its

File: doc/joss-paper/paper.md [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+(right).
+
+## Splitting a Surface from it's Centerlines
```
Comment (lantiga) [inline doc/joss-paper/paper.md]: its :-)

File: doc/joss-paper/paper.md [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+![centerline splitting](splitting.png)
+
+Illustration of the centerline-surface tube containment relationships (left),
+visualization of the surface being split into independent groups based on the
+surfaces membership in a centerline tract/group (right).
+
+## Generating a Volumetric Mesh from a Surface
+
+Tetrahedral, mixed tetrahedral, and boundary layer meshes can be generated from
+a surface and it's centerlines.
```
Comment (lantiga) [inline doc/joss-paper/paper.md]: its

File: doc/joss-paper/paper.md [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+
+``PypeS`` is the glue among vmtk scripts. It allows new scripts to be written
+easily and have a common interface, but, most of all, it allows single vmtk
+scripts to interact with each other, making ``VMTK`` modular and flexible.
+Pypes can be used from the command line, a python interpreter, or from the
+custom ``PypePad`` user interface
+
+![pypepad](pypepad.png)
+
+The VMTK PypePad user interface.
```
Comment (lantiga) [inline doc/joss-paper/paper.md]: Really? PypePad? The swan song uh?

File: doc/joss-paper/paper.md [hunk 7]
```diff
@@ -0,0 +1,154 @@
...
+
+# Acknowledgements & Funding
+
+The Vascular Modeling Toolkit is a community project which is open-source and
+free of charge. We would especially like to recognize significant
+infrastructure contributions from the team at Kitware Inc. (www.kitware.com).
+In addition we would like to acknowledge Orobix Srl. (www.orobix.com) for
+partially funding continued development efforts of ``VMTK``.
+
```
Comment (lantiga) [inline doc/joss-paper/paper.md]: It would be nice to acknowledge the main contributors listed in the source files, if there's room.

Comment (rlizzo) [reply doc/joss-paper/paper.md]: Done!

**Pull request merged**